cypress/e2e/Delete_Function.cy.js

```diff
@@ -0,0 +1,94 @@
+/// <reference types="Cypress"/>
+
+// The Delete story tests is under the assumption you have
+// stories already created with chapters.
+describe('Delete parent story', () => {
+  it('passes', () => {
+    cy.visit("http://localhost:8081")
+    // Login procedures
+    cy.get('input[type="text"]').type('new@email.com')
+    cy.get('input[type="Password"]').type('asdfasdf')
+    cy.get('.v-btn__content').contains('Sign in').should('be.visible').click()
+    cy.pause();
+    // Click the first card title
+    cy.get('.v-card-title.headline').eq(0).click();
+    // Click trash can icon on the story card title
+    cy.get('.mdi-delete').eq(0).click();
```

Comment:
it looks to me like in the code there will be a confirmation dialog asking "are you sure", how is that handled here?  and doe you actually verify in the test that the story selected is deleted?

Reply:
I got this addressed in the recent push! Functionality to show on the confirm window that the story and its attributes are still there after cancelling the confirmation dialog and is removed on confirm.